spec/index.bs

```diff
@@ -1377,10 +1379,16 @@ See the [[#feature-index|Feature Index]] for a description of the functionality
 Each <dfn dfn>limit</dfn> is a numeric limit on the usage of WebGPU on a device.
 
 Each limit has a <dfn dfn for=limit>default</dfn> value.
-Every [=adapter=] is guaranteed to support the default value or [=limit/better=].
-The default is used if a value is not explicitly specified in {{GPUDeviceDescriptor/requiredLimits}}.
 
-One limit value may be <dfn dfn for=limit>better</dfn> than another.
+[=Adapters=] are always guaranteed to support the defaults or [=limit/better=]
+(see [[#adapter capability guarantees]].
+
+[=Devices=] support each limit requested in {{GPUDeviceDescriptor/requiredLimits}},
+or the default if none is requested or the default is [=limit/better=] (see [=a new device=]).
```

Comment:
It feels just a little off to say that it's whatever the descriptor requested. It might be worth saying here that each limit is:
- clamped to be no worse than the default (i.e., in "a new device"), and
- validated to be no better than the adapter's [[limits]]  (i.e., in `requestDevice`).

I don't think you need to say where each rule is imposed, I just mentioned those places to make it clear what I was referring to.

Reply:
Going into detail here is redundant with the parent section "Optional Capabilities", so I've trimmed down both this section and the features section a little bit.